docs-source/content/userguide/managing-domains/_index.md

```diff
@@ -132,3 +134,65 @@ The operator let's you initiate scaling of clusters in various ways:
 * [Using the operator's REST APIs]({{< relref "/userguide/managing-domains/domain-lifecycle/scaling#calling-the-operators-rest-scale-api" >}})
 * [Using WLDF policies]({{< relref "/userguide/managing-domains/domain-lifecycle/scaling#using-a-wldf-policy-rule-and-script-action-to-call-the-operators-rest-scale-api" >}})
 * [Using a Prometheus action]({{< relref "/userguide/managing-domains/domain-lifecycle/scaling#using-a-prometheus-alert-action-to-call-the-operators-rest-scale-api" >}})
+
+### Log files
+
+The operator can automatically override WebLogic domain, server, and introspector `.log` and `.out` locations.
+This occurs if the Domain `logHomeEnabled` field is explicitly set to `true`, or if `logHomeEnabled` isn't set
+and `domainHomeSourceType` is set to `PersistentVolume`.  When overriding, the log location will be the location specified by the `logHome` setting.
+
+If you want to fine tune the `.log` and `.out` rotation behavior for WebLogic domains and servers, then
+you can update the related `Log MBean` in your WebLogic configuration. Alternatively, for WebLogic
+servers, you can set corresponding system properties in `JAVA_OPTIONS`:
+
+- Here are some WLST off-line examples for creating and accessing commonly tuned Log MBeans:
+
+  ```bash
+  # domain log
+  cd('/')
+  create(dname,'Log')
+  cd('/Log/' + dname);
+  
+  # configured server log for a server named 'sname'
+  cd('/Servers/' + sname)
+  create(sname, 'Log')
+  cd('/Servers/' + sname + '/Log/' + sname)
+  
+  # templated (dynamic) server log for a template named 'tname'
+  cd('/ServerTemplates/' + tname)
+  create(tname,'Log')
+  cd('/ServerTemplates/' + tname + '/Log/' + tname)
+  ```
+
+- Here is WLST off-line for commonly tuned Log MBean attributes:
```

Comment:
Here is WLST off-line for commonly tuned Log MBean attributes: -> Here are some WLST offline (command settings OR examples) for commonly tuned Log MBean attributes:

Reply:
fixed